rma/models/rma.py

```diff
@@ -1261,6 +1258,26 @@ def _delivery_group_key(self):
         self.ensure_one()
         return (self.partner_shipping_id.id, self.company_id.id, self.warehouse_id.id)
 
+    def _reception_group_key(self):
+        self.ensure_one()
+        return (self.partner_id.id, self.company_id.id, self.warehouse_id.id)
+
+    def _group_reception_if_needed(self):
```

Comment:
Isn't this better naming?

```suggestion
    def _assign_procurement_group(self):
```

Reply:
I agree, but I wanted to follow the same naming for delivery

https://github.com/OCA/rma/blob/09c8d9154a531806c4e448a5e521b175f226789d/rma/models/rma.py#L1043-L1052

Reply:
Well, OK then.